torchtitan/config_manager.py

```diff
@@ -121,15 +121,16 @@ def __init__(self):
             help="How often to log metrics to TensorBoard, in iterations",
         )
         self.parser.add_argument(
-            "--metrics.enable_color_printing",
-            default=False,
+            "--metrics.enable_tensorboard",
             action="store_true",
-            help="Whether to enable color printing",
+            default=False,
+            help="Whether to log metrics to TensorBoard",
         )
         self.parser.add_argument(
-            "--metrics.enable_tensorboard",
+            "--metrics.enable_color_printing",
             action="store_true",
-            help="Whether to log metrics to TensorBoard",
+            default=True,
```

Comment:
Was flipping the `enable_color_printing` default intended? If so, the action `store_true` does not fit well anymore. It's now true by default